src/pudl/analysis/eia_ferc1_record_linkage.py

```diff
@@ -939,24 +939,31 @@ def check_match_consistency(
     train_df: pd.DataFrame,
     match_set: Literal["all", "overrides"] = "all",
 ) -> pd.DataFrame:
-    """Check how consistent matches are across time.
+    """Check how consistent FERC-EIA matches are with FERC-FERC matches.
+
+    We have two record linkage processes: one that links FERC plant records across time,
+    and another that links FERC plant records to EIA plant-parts. This function checks
+    that the two processes are as consistent with each other as we expect.  Here
+    "consistent" means that each FERC plant ID is associated with a single EIA plant
+    parts ID across time. The reverse is not necessarily required -- a single EIA plant
+    part ID may be associated with various FERC plant IDs across time.
```

Comment:
Looking at the code below I *think* this is right but a second set of 👀 would be helpful.

```py
    count = (
        connects_ferc1_eia[mask]
        .groupby(["plant_id_ferc1"])[["plant_part_id_eia", "capacity_mw_ferc1"]]
        .nunique()
    )
    actual_consistency = len(count[count.plant_part_id_eia == 1]) / len(count)
```

The only material change in here is that the required consistency is 74% rather than 75%.  Everything else is just docs and more explanatory variable names.